web/src/loading.ts

```diff
@@ -91,3 +91,20 @@ export function show_button_spinner($elt: JQuery, using_dark_theme: boolean): vo
     }
     $elt.css("display", "inline-block");
 }
+
+export function show_spinner($element: JQuery, $spinner: JQuery): void {
+    const span_width = $element.find("span").width();
+    const span_height = $element.find("span").height();
```

Comment:
Can we add a class to the span we want to target here? `.find("span")` is not robust code at all.